{"asctime": "2020-03-26 17:21:36,394", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/AssociatedStructure"}
{"asctime": "2020-03-26 17:21:36,406", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StartStructure"}
{"asctime": "2020-03-26 17:21:38,032", "levelname": "DEBUG", "message": "ApplyDataModel :Field StartStructure added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/AssociatedStructure"}
{"asctime": "2020-03-26 17:21:38,033", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EndStructure"}
{"asctime": "2020-03-26 17:21:39,562", "levelname": "DEBUG", "message": "ApplyDataModel :Field EndStructure added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/AssociatedStructure"}
{"asctime": "2020-03-26 17:21:39,562", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Cable', 'geometry': 'POLYLINE', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'cable.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:21:39,826", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Cable"}
{"asctime": "2020-03-26 17:21:41,793", "levelname": "DEBUG", "message": "ApplyDataModel :Cable created"}
{"asctime": "2020-03-26 17:21:41,795", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:41,796", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:21:42,197", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:21:42,198", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:21:42,198", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:21:42,199", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CableId"}
{"asctime": "2020-03-26 17:21:43,792", "levelname": "DEBUG", "message": "ApplyDataModel :Field CableId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:43,793", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RouteId"}
{"asctime": "2020-03-26 17:21:45,410", "levelname": "DEBUG", "message": "ApplyDataModel :Field RouteId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:45,411", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SupportingStructureId"}
{"asctime": "2020-03-26 17:21:47,037", "levelname": "DEBUG", "message": "ApplyDataModel :Field SupportingStructureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:47,037", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Size"}
{"asctime": "2020-03-26 17:21:48,872", "levelname": "DEBUG", "message": "ApplyDataModel :Field Size added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:48,872", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CableType"}
{"asctime": "2020-03-26 17:21:50,783", "levelname": "DEBUG", "message": "ApplyDataModel :Field CableType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:50,784", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SheathId"}
{"asctime": "2020-03-26 17:21:52,582", "levelname": "DEBUG", "message": "ApplyDataModel :Field SheathId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:52,582", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SectionId"}
{"asctime": "2020-03-26 17:21:54,331", "levelname": "DEBUG", "message": "ApplyDataModel :Field SectionId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:54,332", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-26 17:21:56,036", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:56,036", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:21:57,763", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:57,764", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerServicePointId"}
{"asctime": "2020-03-26 17:21:59,556", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerServicePointId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:21:59,556", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeDetailId"}
{"asctime": "2020-03-26 17:22:01,338", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeDetailId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:22:01,339", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RangeLabel"}
{"asctime": "2020-03-26 17:22:03,017", "levelname": "DEBUG", "message": "ApplyDataModel :Field RangeLabel added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Cable"}
{"asctime": "2020-03-26 17:22:03,017", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CableFeature', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'cablefeature.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:22:03,244", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class CableFeature"}
{"asctime": "2020-03-26 17:22:05,254", "levelname": "DEBUG", "message": "ApplyDataModel :CableFeature created"}
{"asctime": "2020-03-26 17:22:05,255", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:05,256", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:22:05,683", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:22:05,684", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:05,685", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:05,687", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:22:07,389", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:07,390", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-26 17:22:09,100", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:09,101", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:22:10,827", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:10,828", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FeatureType"}
{"asctime": "2020-03-26 17:22:12,568", "levelname": "DEBUG", "message": "ApplyDataModel :Field FeatureType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:12,568", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromCableId"}
{"asctime": "2020-03-26 17:22:14,300", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromCableId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:14,300", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToCableId"}
{"asctime": "2020-03-26 17:22:16,010", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToCableId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CableFeature"}
{"asctime": "2020-03-26 17:22:16,010", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CentralOffice', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'centraloffice.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:22:16,234", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class CentralOffice"}
{"asctime": "2020-03-26 17:22:18,328", "levelname": "DEBUG", "message": "ApplyDataModel :CentralOffice created"}
{"asctime": "2020-03-26 17:22:18,329", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOffice"}
{"asctime": "2020-03-26 17:22:18,330", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:22:18,773", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:22:18,774", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:18,774", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:18,776", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-26 17:22:20,483", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOffice"}
{"asctime": "2020-03-26 17:22:20,484", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:22:22,247", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOffice"}
{"asctime": "2020-03-26 17:22:22,248", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CentralOfficeDetail', 'geometry': '', 'spatialref': '', 'fields': 'centralofficedetail.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:22:22,288", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:24,293", "levelname": "DEBUG", "message": "ApplyDataModel :CentralOfficeDetail created"}
{"asctime": "2020-03-26 17:22:24,294", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:24,295", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:22:24,718", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:22:24,719", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:24,720", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:24,722", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-26 17:22:26,133", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:26,133", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Code"}
{"asctime": "2020-03-26 17:22:27,593", "levelname": "DEBUG", "message": "ApplyDataModel :Field Code added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:27,594", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CircuitNumber"}
{"asctime": "2020-03-26 17:22:29,035", "levelname": "DEBUG", "message": "ApplyDataModel :Field CircuitNumber added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:29,036", "levelname": "DEBUG", "message": "ApplyDataModel :Applying DetailId"}
{"asctime": "2020-03-26 17:22:30,505", "levelname": "DEBUG", "message": "ApplyDataModel :Field DetailId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:30,506", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-26 17:22:32,010", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeDetail"}
{"asctime": "2020-03-26 17:22:32,011", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'CentralOfficeArea', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'centralofficearea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:22:32,289", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class CentralOfficeArea"}
{"asctime": "2020-03-26 17:22:34,380", "levelname": "DEBUG", "message": "ApplyDataModel :CentralOfficeArea created"}
{"asctime": "2020-03-26 17:22:34,381", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeArea"}
{"asctime": "2020-03-26 17:22:34,382", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:22:34,813", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:22:34,813", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:34,814", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:34,816", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeAreaId"}
{"asctime": "2020-03-26 17:22:36,490", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeAreaId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeArea"}
{"asctime": "2020-03-26 17:22:36,491", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-26 17:22:38,159", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/CentralOfficeArea"}
{"asctime": "2020-03-26 17:22:38,160", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Equipment', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'equipment.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:22:38,397", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Equipment"}
{"asctime": "2020-03-26 17:22:40,443", "levelname": "DEBUG", "message": "ApplyDataModel :Equipment created"}
{"asctime": "2020-03-26 17:22:40,444", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:40,445", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:22:40,868", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:22:40,868", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:40,869", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:40,870", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:22:42,593", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:42,594", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentId"}
{"asctime": "2020-03-26 17:22:44,276", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:44,276", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentType"}
{"asctime": "2020-03-26 17:22:45,951", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:45,952", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:22:47,721", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:47,721", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-26 17:22:49,417", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:49,418", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RangeLabel"}
{"asctime": "2020-03-26 17:22:51,101", "levelname": "DEBUG", "message": "ApplyDataModel :Field RangeLabel added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:51,102", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Length"}
{"asctime": "2020-03-26 17:22:52,844", "levelname": "DEBUG", "message": "ApplyDataModel :Field Length added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:52,844", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ActualLength"}
{"asctime": "2020-03-26 17:22:54,574", "levelname": "DEBUG", "message": "ApplyDataModel :Field ActualLength added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Equipment"}
{"asctime": "2020-03-26 17:22:54,575", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'MarkupLine', 'geometry': 'POLYLINE', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'markupline.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:22:54,825", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class MarkupLine"}
{"asctime": "2020-03-26 17:22:56,819", "levelname": "DEBUG", "message": "ApplyDataModel :MarkupLine created"}
{"asctime": "2020-03-26 17:22:56,820", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupLine"}
{"asctime": "2020-03-26 17:22:56,821", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:22:57,242", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:22:57,243", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:57,243", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:22:57,245", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:22:58,919", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupLine"}
{"asctime": "2020-03-26 17:22:58,920", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:23:00,662", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupLine"}
{"asctime": "2020-03-26 17:23:00,662", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-26 17:23:02,811", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupLine"}
{"asctime": "2020-03-26 17:23:02,812", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'MarkupPoint', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'markuppoint.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:23:03,056", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class MarkupPoint"}
{"asctime": "2020-03-26 17:23:05,087", "levelname": "DEBUG", "message": "ApplyDataModel :MarkupPoint created"}
{"asctime": "2020-03-26 17:23:05,088", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPoint"}
{"asctime": "2020-03-26 17:23:05,089", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:23:05,527", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:23:05,527", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:05,528", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:05,530", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:23:07,671", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPoint"}
{"asctime": "2020-03-26 17:23:07,672", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:23:09,806", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPoint"}
{"asctime": "2020-03-26 17:23:09,806", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-26 17:23:11,739", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPoint"}
{"asctime": "2020-03-26 17:23:11,740", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'MarkupPolygon', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'markuppolygon.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:23:11,982", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class MarkupPolygon"}
{"asctime": "2020-03-26 17:23:14,017", "levelname": "DEBUG", "message": "ApplyDataModel :MarkupPolygon created"}
{"asctime": "2020-03-26 17:23:14,018", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPolygon"}
{"asctime": "2020-03-26 17:23:14,019", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:23:14,474", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:23:14,474", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:14,475", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:14,477", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:23:16,274", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPolygon"}
{"asctime": "2020-03-26 17:23:16,275", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:23:17,987", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPolygon"}
{"asctime": "2020-03-26 17:23:17,988", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-26 17:23:20,008", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/MarkupPolygon"}
{"asctime": "2020-03-26 17:23:20,009", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'NetworkConnectivity', 'geometry': '', 'spatialref': '', 'fields': 'networkconnectivity.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:23:20,048", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:22,172", "levelname": "DEBUG", "message": "ApplyDataModel :NetworkConnectivity created"}
{"asctime": "2020-03-26 17:23:22,173", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:22,174", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:23:22,633", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:23:22,634", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:22,634", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:22,636", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ConnectionId"}
{"asctime": "2020-03-26 17:23:24,126", "levelname": "DEBUG", "message": "ApplyDataModel :Field ConnectionId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:24,127", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InFeatureId"}
{"asctime": "2020-03-26 17:23:25,558", "levelname": "DEBUG", "message": "ApplyDataModel :Field InFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:25,559", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InConnectionStart"}
{"asctime": "2020-03-26 17:23:26,977", "levelname": "DEBUG", "message": "ApplyDataModel :Field InConnectionStart added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:26,978", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InConnectionStop"}
{"asctime": "2020-03-26 17:23:28,354", "levelname": "DEBUG", "message": "ApplyDataModel :Field InConnectionStop added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:28,354", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OutFeatureId"}
{"asctime": "2020-03-26 17:23:29,845", "levelname": "DEBUG", "message": "ApplyDataModel :Field OutFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:29,845", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OutConnectionStart"}
{"asctime": "2020-03-26 17:23:31,273", "levelname": "DEBUG", "message": "ApplyDataModel :Field OutConnectionStart added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:31,274", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OutConnectionStop"}
{"asctime": "2020-03-26 17:23:32,711", "levelname": "DEBUG", "message": "ApplyDataModel :Field OutConnectionStop added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:32,712", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Loss"}
{"asctime": "2020-03-26 17:23:34,192", "levelname": "DEBUG", "message": "ApplyDataModel :Field Loss added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/NetworkConnectivity"}
{"asctime": "2020-03-26 17:23:34,193", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'PrintTile', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'printtile.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:23:34,437", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class PrintTile"}
{"asctime": "2020-03-26 17:23:36,507", "levelname": "DEBUG", "message": "ApplyDataModel :PrintTile created"}
{"asctime": "2020-03-26 17:23:36,508", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/PrintTile"}
{"asctime": "2020-03-26 17:23:36,509", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:23:36,969", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:23:36,969", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:36,970", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:36,972", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-26 17:23:38,702", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/PrintTile"}
{"asctime": "2020-03-26 17:23:38,703", "levelname": "DEBUG", "message": "ApplyDataModel :Applying PageNumber"}
{"asctime": "2020-03-26 17:23:40,525", "levelname": "DEBUG", "message": "ApplyDataModel :Field PageNumber added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/PrintTile"}
{"asctime": "2020-03-26 17:23:40,525", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Route', 'geometry': 'POLYLINE', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'route.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:23:40,756", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Route"}
{"asctime": "2020-03-26 17:23:42,829", "levelname": "DEBUG", "message": "ApplyDataModel :Route created"}
{"asctime": "2020-03-26 17:23:42,831", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:42,832", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:23:43,341", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:23:43,343", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:43,344", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:43,346", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:23:45,195", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:45,195", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RouteId"}
{"asctime": "2020-03-26 17:23:46,913", "levelname": "DEBUG", "message": "ApplyDataModel :Field RouteId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:46,913", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromStructureId"}
{"asctime": "2020-03-26 17:23:48,600", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromStructureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:48,601", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToStructureId"}
{"asctime": "2020-03-26 17:23:50,337", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToStructureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:50,337", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-26 17:23:52,058", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:52,059", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-26 17:23:53,982", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Route"}
{"asctime": "2020-03-26 17:23:53,983", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'ServicePoint', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'servicepoint.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:23:54,216", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class ServicePoint"}
{"asctime": "2020-03-26 17:23:56,269", "levelname": "DEBUG", "message": "ApplyDataModel :ServicePoint created"}
{"asctime": "2020-03-26 17:23:56,270", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServicePoint"}
{"asctime": "2020-03-26 17:23:56,271", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:23:56,699", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:23:56,700", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:56,700", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:23:56,703", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ServicePointId"}
{"asctime": "2020-03-26 17:23:58,427", "levelname": "DEBUG", "message": "ApplyDataModel :Field ServicePointId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServicePoint"}
{"asctime": "2020-03-26 17:23:58,427", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerNumber"}
{"asctime": "2020-03-26 17:24:00,494", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerNumber added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServicePoint"}
{"asctime": "2020-03-26 17:24:00,495", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CustomerServicePointId"}
{"asctime": "2020-03-26 17:24:02,368", "levelname": "DEBUG", "message": "ApplyDataModel :Field CustomerServicePointId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServicePoint"}
{"asctime": "2020-03-26 17:24:02,368", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-26 17:24:04,073", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServicePoint"}
{"asctime": "2020-03-26 17:24:04,073", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Address"}
{"asctime": "2020-03-26 17:24:05,815", "levelname": "DEBUG", "message": "ApplyDataModel :Field Address added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServicePoint"}
{"asctime": "2020-03-26 17:24:05,815", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'ServingArea', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'servingarea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:24:06,056", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class ServingArea"}
{"asctime": "2020-03-26 17:24:08,068", "levelname": "DEBUG", "message": "ApplyDataModel :ServingArea created"}
{"asctime": "2020-03-26 17:24:08,069", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:08,069", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:24:08,488", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:24:08,488", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:08,489", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:08,490", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:24:10,158", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:10,158", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AreaType"}
{"asctime": "2020-03-26 17:24:11,917", "levelname": "DEBUG", "message": "ApplyDataModel :Field AreaType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:11,918", "levelname": "DEBUG", "message": "ApplyDataModel :Applying EquipmentId"}
{"asctime": "2020-03-26 17:24:13,599", "levelname": "DEBUG", "message": "ApplyDataModel :Field EquipmentId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:13,599", "levelname": "DEBUG", "message": "ApplyDataModel :Applying WorkId"}
{"asctime": "2020-03-26 17:24:15,231", "levelname": "DEBUG", "message": "ApplyDataModel :Field WorkId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:15,232", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SubstationDetailId"}
{"asctime": "2020-03-26 17:24:16,903", "levelname": "DEBUG", "message": "ApplyDataModel :Field SubstationDetailId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:16,904", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-26 17:24:18,845", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:18,846", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ServicePointCount"}
{"asctime": "2020-03-26 17:24:20,757", "levelname": "DEBUG", "message": "ApplyDataModel :Field ServicePointCount added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/ServingArea"}
{"asctime": "2020-03-26 17:24:20,757", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'Structure', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'structure.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:24:20,977", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class Structure"}
{"asctime": "2020-03-26 17:24:22,683", "levelname": "DEBUG", "message": "ApplyDataModel :Structure created"}
{"asctime": "2020-03-26 17:24:22,684", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:22,685", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:24:23,060", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:24:23,061", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:23,061", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:23,063", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:24:24,593", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:24,594", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureId"}
{"asctime": "2020-03-26 17:24:26,194", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:26,195", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SubstationDetailId"}
{"asctime": "2020-03-26 17:24:27,890", "levelname": "DEBUG", "message": "ApplyDataModel :Field SubstationDetailId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:27,891", "levelname": "DEBUG", "message": "ApplyDataModel :Applying StructureType"}
{"asctime": "2020-03-26 17:24:29,607", "levelname": "DEBUG", "message": "ApplyDataModel :Field StructureType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:29,608", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:24:31,266", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:31,266", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromCableId"}
{"asctime": "2020-03-26 17:24:32,974", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromCableId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:32,975", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToCableId"}
{"asctime": "2020-03-26 17:24:34,648", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToCableId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/Structure"}
{"asctime": "2020-03-26 17:24:34,649", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'SupportingStructure', 'geometry': 'POLYLINE', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'supportingstructure.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:24:34,871", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class SupportingStructure"}
{"asctime": "2020-03-26 17:24:36,655", "levelname": "DEBUG", "message": "ApplyDataModel :SupportingStructure created"}
{"asctime": "2020-03-26 17:24:36,656", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:36,657", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:24:37,046", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:24:37,047", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:37,047", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:37,049", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:24:38,698", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:38,699", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RouteId"}
{"asctime": "2020-03-26 17:24:40,308", "levelname": "DEBUG", "message": "ApplyDataModel :Field RouteId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:40,308", "levelname": "DEBUG", "message": "ApplyDataModel :Applying SupportingStructureId"}
{"asctime": "2020-03-26 17:24:41,924", "levelname": "DEBUG", "message": "ApplyDataModel :Field SupportingStructureId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:41,924", "levelname": "DEBUG", "message": "ApplyDataModel :Applying InstallationType"}
{"asctime": "2020-03-26 17:24:43,570", "levelname": "DEBUG", "message": "ApplyDataModel :Field InstallationType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:43,570", "levelname": "DEBUG", "message": "ApplyDataModel :Applying FromDistance"}
{"asctime": "2020-03-26 17:24:45,192", "levelname": "DEBUG", "message": "ApplyDataModel :Field FromDistance added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:45,192", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ToDistance"}
{"asctime": "2020-03-26 17:24:47,467", "levelname": "DEBUG", "message": "ApplyDataModel :Field ToDistance added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:47,469", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Size"}
{"asctime": "2020-03-26 17:24:49,815", "levelname": "DEBUG", "message": "ApplyDataModel :Field Size added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/SupportingStructure"}
{"asctime": "2020-03-26 17:24:49,816", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'WorkArea', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'workarea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:24:50,191", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class WorkArea"}
{"asctime": "2020-03-26 17:24:52,736", "levelname": "DEBUG", "message": "ApplyDataModel :WorkArea created"}
{"asctime": "2020-03-26 17:24:52,737", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:24:52,738", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:24:53,343", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:24:53,344", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:53,344", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:24:53,346", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:24:55,167", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:24:55,168", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AreaCableType"}
{"asctime": "2020-03-26 17:24:56,980", "levelname": "DEBUG", "message": "ApplyDataModel :Field AreaCableType added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:24:56,981", "levelname": "DEBUG", "message": "ApplyDataModel :Applying AreaArchitecture"}
{"asctime": "2020-03-26 17:24:58,715", "levelname": "DEBUG", "message": "ApplyDataModel :Field AreaArchitecture added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:24:58,715", "levelname": "DEBUG", "message": "ApplyDataModel :Applying CentralOfficeId"}
{"asctime": "2020-03-26 17:25:00,530", "levelname": "DEBUG", "message": "ApplyDataModel :Field CentralOfficeId added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:25:00,531", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Name"}
{"asctime": "2020-03-26 17:25:02,417", "levelname": "DEBUG", "message": "ApplyDataModel :Field Name added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:25:02,418", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Colour"}
{"asctime": "2020-03-26 17:25:04,271", "levelname": "DEBUG", "message": "ApplyDataModel :Field Colour added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:25:04,272", "levelname": "DEBUG", "message": "ApplyDataModel :Applying IsActive"}
{"asctime": "2020-03-26 17:25:06,204", "levelname": "DEBUG", "message": "ApplyDataModel :Field IsActive added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:25:06,204", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Status"}
{"asctime": "2020-03-26 17:25:07,986", "levelname": "DEBUG", "message": "ApplyDataModel :Field Status added to C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea"}
{"asctime": "2020-03-26 17:25:07,987", "levelname": "DEBUG", "message": "ApplyDataModel :Applying "}
{"asctime": "2020-03-26 17:25:08,397", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field  failed"}
{"asctime": "2020-03-26 17:25:08,398", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000735: Field Name: Value is required\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:25:08,398", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000735: Field Name: Value is required\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:25:08,401", "levelname": "DEBUG", "message": "ApplyDataModel :Processing feature classes finished."}
{"asctime": "2020-03-26 17:25:56,463", "levelname": "DEBUG", "message": "ApplyDataModel :Setting subtype field C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea,domains failed"}
{"asctime": "2020-03-26 17:25:56,463", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000728: Field domains does not exist within table\nFailed to execute (SetSubtypeField).\n"}
{"asctime": "2020-03-26 17:25:56,464", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 136, in _process_subtypes\n    arcpy.SetSubtypeField_management(full_fc, fd)\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19657, in SetSubtypeField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19654, in SetSubtypeField\n    retval = convertArcObjectToPythonObject(gp.SetSubtypeField_management(*gp_fixargs((in_table, field, clear_value), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000728: Field domains does not exist within table\nFailed to execute (SetSubtypeField).\n"}
{"asctime": "2020-03-26 17:25:56,507", "levelname": "DEBUG", "message": "ApplyDataModel :Setting subtype code C:/Cloud/Dropbox/Work/Axin/rdt.design.datamodel.gdb/WorkArea,domains,1,{'AreaArchitecture': 'WorkAreaArchitecture', 'AreaCableType': 'WorkAreaCableType'} failed"}
{"asctime": "2020-03-26 17:25:56,507", "levelname": "ERROR", "message": "ApplyDataModel :Object: Error in executing tool"}
{"asctime": "2020-03-26 17:25:56,507", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 142, in _process_subtypes\n    arcpy.AddSubtype_management(full_fc, code, st_vals[code])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19589, in AddSubtype\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 19586, in AddSubtype\n    retval = convertArcObjectToPythonObject(gp.AddSubtype_management(*gp_fixargs((in_table, subtype_code, subtype_description), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\nRuntimeError: Object: Error in executing tool"}
{"asctime": "2020-03-26 17:27:51,144", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain ApprovalStatus"}
{"asctime": "2020-03-26 17:27:51,145", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-26 17:27:51,145", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-26 17:27:56,704", "levelname": "DEBUG", "message": "ApplyDataModel :Can't create domain RouteBuildType"}
{"asctime": "2020-03-26 17:27:56,704", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-26 17:27:56,705", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 102, in _process_domains\n    arcpy.CreateDomain_management(self._connection,k,v['description'],v['type'],v['domaintype'],v['split'],v['merge'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1865, in CreateDomain\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 1862, in CreateDomain\n    retval = convertArcObjectToPythonObject(gp.CreateDomain_management(*gp_fixargs((in_workspace, domain_name, domain_description, field_type, domain_type, split_policy, merge_policy), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nERROR 000192: Invalid value for Domain Name\nFailed to execute (CreateDomain).\n"}
{"asctime": "2020-03-26 17:27:58,373", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlineChangeArea', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'redlinechangearea.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:27:58,613", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlineChangeArea exists."}
{"asctime": "2020-03-26 17:27:58,614", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-26 17:28:00,297", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class RedlineChangeArea"}
{"asctime": "2020-03-26 17:28:02,550", "levelname": "DEBUG", "message": "ApplyDataModel :RedlineChangeArea created"}
{"asctime": "2020-03-26 17:28:02,551", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineChangeArea"}
{"asctime": "2020-03-26 17:28:02,552", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Description"}
{"asctime": "2020-03-26 17:28:04,355", "levelname": "DEBUG", "message": "ApplyDataModel :Field Description added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineChangeArea"}
{"asctime": "2020-03-26 17:28:04,355", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comments"}
{"asctime": "2020-03-26 17:28:06,176", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comments added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineChangeArea"}
{"asctime": "2020-03-26 17:28:06,177", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ApprovalStatus"}
{"asctime": "2020-03-26 17:28:07,928", "levelname": "DEBUG", "message": "ApplyDataModel :Field ApprovalStatus added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineChangeArea"}
{"asctime": "2020-03-26 17:28:07,929", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ChangeId"}
{"asctime": "2020-03-26 17:28:09,501", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field ChangeId failed"}
{"asctime": "2020-03-26 17:28:09,502", "levelname": "ERROR", "message": "ApplyDataModel :ERROR 000356: Invalid domain type.\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:09,502", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: ERROR 000356: Invalid domain type.\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:09,504", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:28:09,881", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:28:09,882", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:09,882", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:09,884", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RedlineFeatureId"}
{"asctime": "2020-03-26 17:28:11,558", "levelname": "DEBUG", "message": "ApplyDataModel :Field RedlineFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineChangeArea"}
{"asctime": "2020-03-26 17:28:11,558", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlinePointMarker', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'redlinepointmarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-26 17:28:11,736", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlinePointMarker exists."}
{"asctime": "2020-03-26 17:28:11,737", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-26 17:28:13,065", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class RedlinePointMarker"}
{"asctime": "2020-03-26 17:28:14,849", "levelname": "DEBUG", "message": "ApplyDataModel :RedlinePointMarker created"}
{"asctime": "2020-03-26 17:28:14,850", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlinePointMarker"}
{"asctime": "2020-03-26 17:28:14,851", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-26 17:28:16,461", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlinePointMarker"}
{"asctime": "2020-03-26 17:28:16,462", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comments"}
{"asctime": "2020-03-26 17:28:18,554", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comments added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlinePointMarker"}
{"asctime": "2020-03-26 17:28:18,555", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:28:19,540", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:28:19,540", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:19,541", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:19,543", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RedlineFeatureId"}
{"asctime": "2020-03-26 17:28:21,447", "levelname": "DEBUG", "message": "ApplyDataModel :Field RedlineFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlinePointMarker"}
{"asctime": "2020-03-26 17:28:21,447", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlineLineMarker', 'geometry': 'POLYLINE', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'redlinelinemarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:28:21,671", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlineLineMarker exists."}
{"asctime": "2020-03-26 17:28:21,672", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-26 17:28:23,467", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class RedlineLineMarker"}
{"asctime": "2020-03-26 17:28:25,592", "levelname": "DEBUG", "message": "ApplyDataModel :RedlineLineMarker created"}
{"asctime": "2020-03-26 17:28:25,593", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineLineMarker"}
{"asctime": "2020-03-26 17:28:25,594", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-26 17:28:27,249", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineLineMarker"}
{"asctime": "2020-03-26 17:28:27,250", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comments"}
{"asctime": "2020-03-26 17:28:28,811", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comments added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineLineMarker"}
{"asctime": "2020-03-26 17:28:28,811", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:28:29,257", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:28:29,257", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:29,257", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:29,259", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RedlineFeatureId"}
{"asctime": "2020-03-26 17:28:30,720", "levelname": "DEBUG", "message": "ApplyDataModel :Field RedlineFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineLineMarker"}
{"asctime": "2020-03-26 17:28:30,721", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'RedlineComment', 'geometry': '', 'spatialref': '', 'fields': 'redlinecomment.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': False}"}
{"asctime": "2020-03-26 17:28:30,901", "levelname": "DEBUG", "message": "ApplyDataModel :Feature class RedlineComment exists."}
{"asctime": "2020-03-26 17:28:30,902", "levelname": "DEBUG", "message": "ApplyDataModel :We have clearance to delete Clarence."}
{"asctime": "2020-03-26 17:28:31,958", "levelname": "DEBUG", "message": "ApplyDataModel :Creating table RedlineComment"}
{"asctime": "2020-03-26 17:28:33,681", "levelname": "DEBUG", "message": "ApplyDataModel :RedlineComment created"}
{"asctime": "2020-03-26 17:28:33,682", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineComment"}
{"asctime": "2020-03-26 17:28:33,682", "levelname": "DEBUG", "message": "ApplyDataModel :Applying ApprovalStatus"}
{"asctime": "2020-03-26 17:28:35,003", "levelname": "DEBUG", "message": "ApplyDataModel :Field ApprovalStatus added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineComment"}
{"asctime": "2020-03-26 17:28:35,004", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comment"}
{"asctime": "2020-03-26 17:28:36,447", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comment added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineComment"}
{"asctime": "2020-03-26 17:28:36,448", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:28:36,940", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:28:36,941", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:36,941", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:28:36,943", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RedlineFeatureId"}
{"asctime": "2020-03-26 17:28:38,632", "levelname": "DEBUG", "message": "ApplyDataModel :Field RedlineFeatureId added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineComment"}
{"asctime": "2020-03-26 17:28:38,633", "levelname": "DEBUG", "message": "ApplyDataModel :Applying RedlineCommentId"}
{"asctime": "2020-03-26 17:28:40,407", "levelname": "DEBUG", "message": "ApplyDataModel :Field RedlineCommentId added to C:/Cloud/Dropbox/Work/Axin/rdt.markup.datamodel.gdb/RedlineComment"}
{"asctime": "2020-03-26 17:28:40,408", "levelname": "DEBUG", "message": "ApplyDataModel :Processing feature classes finished."}
{"asctime": "2020-03-26 17:29:01,158", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'FieldingAreaMarker', 'geometry': 'POLYGON', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'fieldingareamarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
